examples/external-api-request/external_api_request.py

```diff
@@ -14,6 +15,8 @@
 
 from utils import run_golem_example
 
+event = asyncio.Event()
```

Comment:
I would rename it to sth more meaningful like: "task_finished_event" or sth similar

Reply:
It would be controversial because it was supposed to be a service API example, not a task API (even if it looks more like a task).
Renamed it to `service_finished_event`.